managers/bosh-manager/DirectorService.js

```diff
@@ -778,7 +768,13 @@ class DirectorService extends BaseDirectorService {
     return Promise.join(
         this.executeActions(CONST.SERVICE_LIFE_CYCLE.PRE_BIND, actionContext),
         this.getDeploymentIps(deploymentName),
-        (preBindResponse, ips) => this.agent.createCredentials(ips, binding.parameters, preBindResponse)
+        (preBindResponse, ips) => utils.retry(() => this.agent.createCredentials(ips, binding.parameters, preBindResponse), {
```

Comment:
Agent.js does a retry and here you are retrying thrice? so in effect you have max 6 attempts at recreating creds. Can you add retry only at one place? Just having it here as well is fine.

Reply:
removed from agent.

Reply:
Ok.